package.json

```diff
@@ -32,6 +32,7 @@
   "license": "MIT",
   "dependencies": {
     "chalk": "^2.2.0",
+    "core-validate-commit": "^3.5.0",
```

Comment:
I don't think we should include it as a dependency if we are using npx. It would download it automatically.

Reply:
Oops i missed this..., and landed this maybe do a followup.

Reply:
I prefer to have as a dependency. Otherwise it's slower and inefficient (always reinstalling the package)

Reply:
then what is the benefit of using npx?

Reply:
I think i this case it was to avoid adding `.cmd` to script in `./node_modules/,bin/core-validate-commit` adn writing out the path to script.

Reply:
Yes, it was my intention but I think I was wrong. The command is executed from the `node` directory so it cannot see `core-validate-commit`.